### PR TITLE
Allow image preview

### PR DESCRIPTION
**Pull request by rbatistadev**
Allow image preview when downloadPath exists

### CODE REVIEW

File: library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx [hunk 1]
```diff
@@ -30,6 +33,33 @@ export const ImageFileUploader: FileUploaderType = (
   const fileSize = fileValue?.file ? fileValue.file?.size : fileValue?.fileSize;
   const fileSizeMb = Math.round(((fileSize || 0) / 1024 / 1024) * 10) / 10;
 
+  const apiDownload = useStoreActions((actions) => {
+    return actions.api.download;
+  });
+  const [imageSrc, setImageSrc] = useState('' as string | null);
```
Comment (mmadariaga) [inline library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx:39]:
```suggestion
  const [imageSrc, setImageSrc] = useState<string | null>(null);
```

File: library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx [hunk 2]
```diff
@@ -30,6 +33,33 @@ export const ImageFileUploader: FileUploaderType = (
...
+          path: downloadPath as string,
+          params: {},
+          successCallback: async (data: Record<string, any> | Array<Record<string, any>> | Blob) => {
+            setImageSrc(URL.createObjectURL(data as Blob))
+          },
+        });
+      }
+    } finally {
+      setImageSrc(null);
+    }
```
Comment (mmadariaga) [inline library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx:60]:
```suggestion
     setImageSrc(null);

      if (fileValue.file) {
        const reader = new FileReader();
        reader.onloadend = () => {
            setImageSrc(reader.result as string);
        };
        reader.readAsDataURL(fileValue.file);
        
        return;
      }

       apiDownload({
          path: downloadPath as string,
          params: {},
          successCallback: async (data: Record<string, any> | Array<Record<string, any>> | Blob) => {
            setImageSrc(URL.createObjectURL(data as Blob))
          },
       });

```
Would be easier to read / understand

File: library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx [hunk 3]
```diff
@@ -59,7 +89,11 @@ export const ImageFileUploader: FileUploaderType = (
 
         {fileName && (
           <StyledImageContainer className={disabled ? 'disabled' : ''}>
-            {values.id && <AccountCircleIcon onClick={handleDownload} />}
+            {
+              imageSrc
+                ? (<StyledImagePreview src={imageSrc} />)
```
Comment (mmadariaga) [inline library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx:94]: Are we losing download feature?

**Review by mmadariaga — CHANGES_REQUESTED** [review]

File: library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx (hunk 3 above)
Comment (rbatistadev) [reply library/src/services/form/Field/FileUploader/Variant/ImageFileUploader.tsx:94]: yep, I'll correct!

File: library/package.json [hunk 4]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "@irontec-voip/ivoz-ui",
+  "name": "@irontec/ivoz-ui",
   "version": "1.3.15",
```
Comment (mmadariaga) [inline library/package.json:3]:
```suggestion
  "version": "1.3.16",
```

**Review by mmadariaga — APPROVED** [review]

**Pull request merged**